tests/k6/scenarios/infer_constant_rate.js

```diff
@@ -47,6 +53,16 @@ export default function (config) {
     } else {
         doInfer(modelName, modelNameWithVersion, config, false, idx)
     }
+
+    // for simplicity we only change model replicas in the first VU
```

Comment:
Could we move this into a function in `utils.js` (receiving config as an argument) and call it from both places? I'll probably want to reuse this piece of code in other scenarios as well.

I know in general one might one to customise this on their own load scenario (so reusing the common code will not be feasible all the time), but it would be good to have something to call for a default replica change feature that reads the existing config environment variables.

Reply:
fixed in [c3d1c77](https://github.com/SeldonIO/seldon-core/pull/5977/commits/c3d1c7732f4e11890f26368cbceaae8a8192e976)